cmd/charm/charmcmd/pull.go

```diff
@@ -46,8 +46,7 @@ will fetch the wordpress charm into the
 directory "wordpress" in the current directory.
 
 When a specific charm or bundle revision is provided,
-the channel parameter is ignored. Otherwise the "stable"
-channel is used by default. To select another channel,
+the channel parameter is ignored. To select a channel,
```

Comment:
I wonder if we should have one canonical description for the behaviour of the channel flag and refer to it in each doc, rather than duplicating the same text everywhere.